pkg/typesv1beta1/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package typesv1beta1 defines v1beta1 structures for installer configuration and
```

Comment:
Yes, we need to structure `pkg/types` better to handle support for previous versions. I did not want to tackle that as part of this PR. Again, the support for v1beta1 types is only temporary so that this PR will not break people immediately. The `pkg/types/v1beta1' will be removed shortly after this PR lands.